src/frontend/src/helpers/Storage/LocalStorage.ts

```diff
@@ -56,9 +59,10 @@ class LocalStorage implements AbstractStorage {
     * @param {string} file_history: The history of the file.
     * @param {string || any false value} checksum : The online checksum of the file, false to not update (offline write).
     */
-  public async writeFile(fid: FileID, contents?: string): Promise<void> {
```

Comment:
writeFile will be called with contents set to undefined in syncAll -- that's the backend letting the frontend know that

1)  the file exists, but
2)  isn't available offline (due to being a binary file, etc...)